docs/apis/_files/pix-dir.mdx

```diff
@@ -0,0 +1,5 @@
+Plugins can provide icons in several formats, and most plugin types require that a default icon be provided.
+
+Where a browser supports it, the `svg` format is used, falling back to `png` formats when an SVG is unavailable.
+
+Full details of the correct naming, sizing, and design guidelines for icons in Moodle can be found in the [[Moodle icons]] documentation.
```

Comment:
The https://docs.moodle.org/dev/Moodle_icons is pointing to a project page which, from 4.0 onwards, is kind of deprecated... So probably we should create a separate page for that at any point (it's out of the scope of this issue, so maybe it worth creating a separate issue to not forget about it).

Reply:
Yeah - I agree.  I've created #82